src/Config.ts

```diff
@@ -150,36 +147,6 @@ export interface ConfigParams {
    */
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   functionPlugins: any[],
-  /**
-   * A GPU.js constructor used by array functions.
-   *
-   * When not provided, the plain CPU implementation is used.
-   *
-   * @deprecated since version 1.2.
-   *
-   * @default undefined
-   *
-   * @category Engine
-   */
-  gpujs?: any,
-  /**
-   * Sets array calculations to use either GPU or CPU.
-   *
-   * When set to `gpu`, tries to use GPU for array calculations.
-   *
-   * When set to `cpu`, enforces CPU usage.
-   *
-   * Use other values only for debugging purposes.
-   *
-   * For more information, see the [GPU.js documentation](https://github.com/gpujs/gpu.js/#readme).
-   *
-   * @deprecated since version 1.2
-   *
-   * @default 'gpu'
-   *
-   * @category Engine
-   */
-  gpuMode: GPUMode,
```

Comment:
These config options are marked as deprecated since version `1.2`. Should we keep them here for backward compatibility (they'd have no effect) or can they be removed?